packages/cli/oclif.manifest.json

```diff
@@ -1 +1 @@
\ No newline at end of file
+{"version":"3.35.0","commands":{"editions":{"id":"editions","description":"Shopify editions","strict":true,"pluginName":"@shopify/cli","pluginAlias":"@shopify/cli","pluginType":"core","hidden":true,"aliases":[],"flags":{},"args":[]},"kitchen-sink":{"id":"kitchen-sink","description":"View all the available UI kit components","strict":true,"pluginName":"@shopify/cli","pluginAlias":"@shopify/cli","pluginType":"core","hidden":true,"aliases":[],"flags":{},"args":[]},"upgrade":{"id":"upgrade","description":"Upgrade the Shopify CLI.","strict":true,"pluginName":"@shopify/cli","pluginAlias":"@shopify/cli","pluginType":"core","aliases":[],"flags":{"path":{"name":"path","type":"option","description":"The path to your project directory.","hidden":false,"multiple":false}},"args":[]},"version":{"id":"version","description":"Shopify CLI version.","strict":true,"pluginName":"@shopify/cli","pluginAlias":"@shopify/cli","pluginType":"core","aliases":[],"flags":{},"args":[]},"auth:logout":{"id":"auth:logout","description":"Logout from Shopify.","strict":true,"pluginName":"@shopify/cli","pluginAlias":"@shopify/cli","pluginType":"core","aliases":[],"flags":{},"args":[]},"kitchen-sink:async":{"id":"kitchen-sink:async","description":"View the UI kit components that process async tasks","strict":true,"pluginName":"@shopify/cli","pluginAlias":"@shopify/cli","pluginType":"core","aliases":[],"flags":{},"args":[]},"kitchen-sink:banners":{"id":"kitchen-sink:banners","description":"View the UI kit components that display banners","strict":true,"pluginName":"@shopify/cli","pluginAlias":"@shopify/cli","pluginType":"core","aliases":[],"flags":{},"args":[]},"kitchen-sink:prompts":{"id":"kitchen-sink:prompts","description":"View the UI kit components prompts","strict":true,"pluginName":"@shopify/cli","pluginAlias":"@shopify/cli","pluginType":"core","aliases":[],"flags":{},"args":[]}}}
```

Comment:
No idea why those are different ? Upstream problem in [ink-commands-implementation](https://github.com/Shopify/cli/tree/ink-commands-implementation) maybe?

Reply:
If that's the case, I'll change the changeset.